### PR TITLE
Support for populating and reading db table of Daklapack "articles"

### PR DESCRIPTION
**Pull request by AmandaBirmingham**
A bite-sized chunk of the new daklapack order functionality :)

### CODE REVIEW

File: microsetta_private_api/db/patches/0071.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
+-- add and fill daklapack_article table
+
+-- create table barcodes.daklapack_article
+CREATE TABLE barcodes.daklapack_article
+(
+    dak_article_id uuid DEFAULT uuid_generate_v4() NOT NULL,
+    dak_article_code INTEGER NOT NULL,
```
Comment (dhakim87) [inline microsetta_private_api/db/patches/0071.sql:7]: Is article code also unique?  Should it have another unique index on it?

Comment (dhakim87) [reply microsetta_private_api/db/patches/0071.sql:7]: Should be indexed regardless as it appears in the ORDER BY clause

**Review by dhakim87 — APPROVED** [review]
Seems good - maybe add a (possibly unique) index to article code for the order by clause.

File: microsetta_private_api/db/patches/0071.sql (hunk 1 above)
Comment (AmandaBirmingham) [reply microsetta_private_api/db/patches/0071.sql:7]: Excellent--done!

**Pull request merged**